docs/kyma-environment-broker/08-07-swagger.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Check API using swagger
+type: Tutorials
```

Comment:
I'm not sure Tutorials is the right type here since it's not really a step-by-step instruction 🤔 I'd rather classify this as a Details doc.